src/main/msp/msp.c

```diff
@@ -3216,6 +3219,11 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
             currentPidProfile->tpa_rate = MIN(sbufReadU8(src), TPA_MAX);
             currentPidProfile->tpa_breakpoint = sbufReadU16(src);
         }
+        if (sbufBytesRemaining(src) >= 3) {
+            // Added with API 1.46: dono if this is legit, TBC
+            currentPidProfile->tpa_rate_lower = MIN(sbufReadU8(src), TPA_MAX);
+            currentPidProfile->tpa_breakpoint_lower = sbufReadU16(src);
+        }
 
```

Comment:
This is a new feature so these parameters shouldn't be added to msp for 4.5. Parameters can be added in the next release when it has been tested and we're sure it's something we want to keep

Reply:
Sure, should I just revert all the changes in msp.c?

Reply:
yes, thanks 🙂

Reply:
Will do ASAP :-)